crates/primitives/fee/src/lib.rs

```diff
@@ -22,13 +22,15 @@ use blockifier::transaction::transaction_utils::{calculate_l1_gas_usage, calcula
 #[cfg(not(feature = "std"))]
 use hashbrown::HashMap;
 use mp_state::StateChanges;
+use serde::Deserialize;
```

Comment:
Isn't `serde` optional?
I think if you `cd crates/primitives/fee && cargo check` a compilation error will be raised

Reply:
Corrected it